node/sc/bridge_manager.go

```diff
@@ -1143,3 +1161,23 @@ func (bm *BridgeManager) GetFeeReceiver(bridgeAddr common.Address) (common.Addre
 
 	return bi.bridge.FeeReceiver(nil)
 }
+
+// IsInParentAddrs returns true if the bridgeAddr is in the list of parent bridge addresses and returns false if not.
+func (bm *BridgeManager) IsInParentAddrs(bridgeAddr common.Address) bool {
+	return bm.IsValidBridgeAddr(bridgeAddr, bm.GetAllParentBridgeAddrs())
```

Comment:
```suggestion
	for _, journal := range bm.journal.cache {
		if journal.ChildAddress == bridgeAddr {
			return true
		}
	}
	return false
```

something like this?

Reply:
1. Okay, then how about making it as a function? 
2. The functions `GetAllChildbridgeAddrs()` and `GetAllParentBridgeAddrs` are now not called if your change is reflected. I think leaving them as seems okay. What do you think?

Reply:
1. I think it is already a function?
2. yes. removing them is good to me.

Reply:
I'm sorry. I mixed up that logic is in `sub_bridge_handler.go`. Updated with a simplified one.